src/en/alternativescans/src/eu/kanade/tachiyomi/extension/en/alternativescans/AlternativeScans.kt

```diff
@@ -0,0 +1,176 @@
+package eu.kanade.tachiyomi.extension.en.alternativescans
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.Headers
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class AlternativeScans : HttpSource() {
+
+    override val name = "Alternative Scans"
+    override val baseUrl = "https://alternativescans.icu"
+    private val apiUrl = "https://api.alternativescans.icu"
+    override val lang = "en"
+    override val supportsLatest = true
+
+    private val json = Json { ignoreUnknownKeys = true; isLenient = true }
+
+    private val apiHeaders: Headers by lazy {
+        Headers.Builder()
+            .add("Referer", baseUrl)
+            .build()
+    }
+
+    // Popular
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$apiUrl/api/admin/getLatestUpdate", apiHeaders)
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val result = json.decodeFromString<ApiResponse>(response.body.string())
+        val mangas = result.latestReleases
+            .distinctBy { it.manga }
+            .map {
+                SManga.create().apply {
+                    title = it.title
+                    thumbnail_url = it.seriesThumbnail
+                    url = "/manga/${it.nick}?id=${it.manga}"
+                }
+            }
+        return MangasPage(mangas, false)
+    }
+
+    // Latest
+    override fun latestUpdatesRequest(page: Int): Request = popularMangaRequest(page)
+
+    override fun latestUpdatesParse(response: Response): MangasPage = popularMangaParse(response)
```

Comment:
set `supportsLatest` to false, as popular and latest shouldn't be same. then throw from these methods